snap/README.md

```diff
@@ -69,40 +63,44 @@ ensures that as well as starting the service now, it will be automatically start
 $ sudo snap start --enable edgex-device-rfid-llrp.device-rfid-llrp
 ```
 
-### Using a content interface to set device configuration
-
-The `device-config` content interface allows another snap to seed this device
-snap with configuration files under the `$SNAP_DATA/config/device-rfid-llrp/res` directory.
+## Subnet setup
 
-Note that the `device-config` content interface does NOT support seeding of the Secret Store Token because that file is expected at a different path.
+The `DiscoverySubnets` setting needs to be provided before a device discovery can occur. This can be done manually or using a script to run auto-configure:
 
-To use, create a new snap with a directory containing the configuration files.
-Your `snapcraft.yaml` file then needs to define a slot with read access to the directory you are sharing.
+1. Using `snap set` to set your local subnet information. Example:
 
+```bash
+$ sudo snap set edgex-device-rfid-llrp env.app-custom.discovery-subnets="192.168.10.0/24"
+curl -X POST http://localhost:59989/api/v2/discovery
```

Comment:
Please reference the appropriate Github issue here and mention that this will work only after the issue is resolved.

The issue is now in app-functions-sdk, which isn't even used by this service: https://github.com/edgexfoundry/app-functions-sdk-go/issues/1043

Reply:
updated - please see latest commit.